integrations/azure/steps/complete-ort-run.yml

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2020 Bosch.IO GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+# This file contains a complete ort run: analyze - scan - evaluate - report
+
+parameters:
+  projectVcsUrl: ''
+  projectVcsRevision: 'master'
+  useCredentials: 'false'
+  projectCredentialTemplate: ''
+
+  ortProjectVcsUrl: 'none'
+  ortProjectVcsRevision: 'master'
+  ortUseCredentials: 'false'
+  ortConfigCredentialTemplate: ''
+
+  logLevel: 'default'
+  stacktrace: 'false'
+  allowDynamicVersions: 'false'
+  useClearlyDefined: 'true'
```

Comment:
Just asking for a confirmation : Are we supposed to always use Clearly Defined ?

Reply:
Good point, these two options should be configurable via job parameters.

Reply:
This is configurable. The values here are just "defaults". I could make them blank to make it simpler to understand.

Reply:
I see. 
I think we need to keep the default values for ort-pipeline.yml and complete-ort-run.yml.
However maybe "false" is a saner default value for Clearly Defined ?

Reply:
We should align with the default values in `Jenkinsfile`, but it seems that is the case already:

https://github.com/oss-review-toolkit/ort/blob/9f1f736a416ef1cc4f34e798d2d1e05a18a088eb/integrations/Jenkinsfile#L105-L119

Reply:
Yes, I initially aligned the pipeline variables with the Jenkinsfile.